stdpopsim/catalog/GorGor/demographic_models.py

```diff
@@ -0,0 +1,260 @@
+import msprime
+import stdpopsim
+
+_species = stdpopsim.get_species("GorGor")
+
+###########################################################
+#
+# Demographic models
+#
+###########################################################
+
+_pawar2023 = stdpopsim.Citation(
+    author="Pawar et al. 2023",
+    year=2023,
+    doi="https://doi.org/10.1038/s41559-023-02145-2",
+    reasons={stdpopsim.CiteReason.DEM_MODEL},
+)
+
+
+def _gorilla_ghost():
+    id = "GorillaGhost_5P23"
+    description = "Ghost admixture in eastern gorillas"
+    long_description = """
+        Demographic model of ghost admixture into eastern gorillas
+        from Pawar et al. (2023) Fig. 2A and Supp. Table 1.
+        This model simulates five populations:
+        mountain gorillas, eastern lowland gorillas, western lowland gorillas,
+        cross river gorillas, and a extinct ghost lineage. The ghost admixture
+        event is modelled as a 2.47% pulse from the ghost lineage to a
+        population ancestral to eastern gorillas. Migration events among
+        eastern and western lowland gorillas are modelled as single generation
+        pulses. Population size changes are also modelled.
+    """
+
+    populations = [
+        stdpopsim.Population(
+            id="cross_river", description="Contemporary Cross River Gorillas"
+        ),
+        stdpopsim.Population(
+            id="western_lowland", description="Contemporary Western Lowland Gorillas"
+        ),
+        stdpopsim.Population(
+            id="eastern_lowland", description="Contemporary Eastern Lowland Gorillas"
+        ),
+        stdpopsim.Population(
+            id="mountain", description="Contemporary Mountain Gorillas"
+        ),
+        stdpopsim.Population(
+            id="ghost", description="Extinct ghost lineage", sampling_time=None
+        ),
+    ]
+
+    citations = [_pawar2023]
+    generation_time = 19
+    mutation_rate = 1.235e-8
+
+    N_cross_river = 14558
+    N_western_lowland = 64749
+    N_eastern_lowland = 20894
+    N_mountain = 2158
+
+    # The effective population size of the extinct ghost population is
+    # difficult to estmiate due to the low admixture proportion into Eastern
+    # Gorillas. Here, it is somewhat arbitrarly set to 25,000 individuals.
```

Comment:
You'll want to make this clear in the QC issue, so whoever QCs it can follow along.